CONTRIBUTING.md

```diff
@@ -19,23 +19,36 @@ _Note: replace "your-username" with your GitHub handle_
 Install all package dependencies and link local packages:
 
 ```
-yarn bootstrap
+theme-scripts $ yarn bootstrap
 ```
 
 Write some features. Run the tests with:
 
 ```
-yarn test
+theme-scripts $ yarn test
+```
+
+Adding dependency to all packages
+
+```
+theme-scripts $ yarn lerna add [package-name]
+```
+
+Adding dependency to a specific package
+
+```
+theme-scripts $ yarn lerna add [package-name] --scope=@shopify/theme-[name]
```

Comment:
Is this just adding a new themes package dependency into another themes package? If it is, let's not lump this with the startup / testing instructions.

## Linking a new themes package with Lerna

Add new themes package to all themes package

```
theme-scripts $ yarn lerna add [new-themes-package-name]
```

Add new themes package to a specific themes package

```
theme-scripts $ yarn lerna add [new-themes-package-name] --scope=@shopify/theme-[name]
```

Reply:
Fair enough. I'll remove that for now.